configure

```diff
@@ -3790,7 +3790,7 @@ if test "$_sdlnet" = auto ; then
 	if test "$_sdl" = no || test "$_sdlMajorVersionNumber" = 2; then
 		if test "$_pkg_config" = "yes" && $_pkgconfig --exists SDL2_net; then
 			append_var SDL2_NET_LIBS "`$_pkgconfig --libs SDL2_net`"
-			append_var SDL2_NET_CFLAGS "`$_pkgconfig --cflags SDL2_net`"
+			append_var SDL2_NET_CFLAGS "`$_pkgconfig --cflags SDL2_net | sed 's/[[:space:]]*-Dmain=SDL_main//g'`"
```

Comment:
Is it really needed here?

Reply:
Yes, because SDL_net "requires" sdl2, so it returns its flags too.